spec/unit/controllers/runtime/spaces_controller_spec.rb

```diff
@@ -1269,120 +1270,23 @@ def decoded_guids
       let(:isolation_segment_model) { IsolationSegmentModel.make }
       let(:organization) { Organization.make }
       let(:space) { Space.make(organization: organization) }
+      let(:assigner) { IsolationSegmentAssign.new }
 
       context 'associating an isolation_segment' do
-        context 'when the space contains no apps' do
```

Comment:
@tharrisca What is the difference in the space having apps or not when assigning an Isolation Segment? If the user is unable to perform the action, then they will get a 403 regardless. Those tests are still in this `spaces_controller_spec.rb` file.